File: VB/DXWebApplication1/Controllers/HomeController.vb
Imports System
Imports System.Web.Mvc

Namespace DXWebApplication1.Controllers
	Public Class HomeController
		Inherits Controller

		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Function GridViewPartial() As ActionResult
			Return PartialView(DataProvider.GetDemoModel())
		End Function
		Public Function ComboBoxPartial() As ActionResult
			Return PartialView(DataProvider.GetDemoModel())
		End Function
	End Class
End Namespace

File: CS/DXWebApplication1/Controllers/HomeController.cs
using System;
using System.Web.Mvc;

namespace DXWebApplication1.Controllers {
    public class HomeController: Controller {
        public ActionResult Index() {
            return View();
        }
        public ActionResult GridViewPartial() {
            return PartialView(DataProvider.GetDemoModel());
        }
        public ActionResult ComboBoxPartial() {
            return PartialView(DataProvider.GetDemoModel());
        }
    }
}